Plots/Satellites/NCL_sat_3.py

```diff
@@ -0,0 +1,155 @@
+"""
+NCL_sat_3.py
+================
+
+This script illustrates the following concepts:
+    - zooming into an orthographic projection
+    - plotting filled contour data on an orthographic map
+    - plotting lat/lon tick marks on an orthographic map
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/sat_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/sat_3_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import xarray as xr
+import numpy as np
+import matplotlib.ticker as mticker
+
+import geocat.datafiles as gdf
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Define a helper function for plotting lat/lon ticks on an orthographic plane
+
+
+def plotOrthoTicks(coords, loc):
```

Comment:
In #150 on lines 98-105 of `NCL_shapefiles_1.py` I had a more succinct way of drawing latitude and longitude ticks on a Lambert Conformal projection . I don't know if it will work for Orthographic, but you might be able to modify it for a solution that is shorter than this. If not, what you have is fine.

Reply:
I tried this, but there is no way to specify which ticks you want labeled, or where you want the labels to be. It just plots labels automatically based on which gridlines you specified. Do you think it would be better if I just used this built-in function even if the resulting plot doesn't match the NCL version? This is what it looks like:
![sat_3 alt ticks](https://user-images.githubusercontent.com/36773444/87082312-a480e580-c1df-11ea-90b9-a69d3ed7f021.PNG)

Reply:
I agree with Rose that what you had before should be fine. It looks like cartopy did try to [add support](https://github.com/SciTools/cartopy/pull/1117) for adding lat/lon lines for all projections, but its pretty basic in the way it will be projected and I am assuming thats you used for this one. I think overall, it may be best to show off the capabilities of python over just using built-in functions when necessary. My impression is the examples we produce won't just be used for simply showing projections but also as a tool to show how to use Python for anyone coming over from NCL exclusively.